ush/yaml_from_parm.sh

```diff
@@ -18,7 +18,7 @@ if [[ "$1" == "jedivar" ]]; then
 
 else
   sed -e "s/@analysisDate@/${analysisDate}/" -e "s/@beginDate@/${beginDate}/" \
-    "${EXPDIR}/config/getkf_${TYPE}.yaml" > getkf.yaml
+    "${PARMrrfs}/getkf_${TYPE}.yaml" > getkf.yaml
```

Comment:
Same as the above.  
We intentionally use `${EXPDIR}/config/` here.
See PR https://github.com/NOAA-EMC/rrfs-workflow/pull/745

Reply:
changed